examples/fabric/v4/cloudRouterConnectivity/routing-protocol-bgp/terraform.tfvars.example

```diff
@@ -1,9 +1,11 @@
-equinix_client_id      = "MyEquinixClientId"
-equinix_client_secret  = "MyEquinixSecret"
+equinix_client_id     = "MyEquinixClientId"
+equinix_client_secret = "MyEquinixSecret"
 
-connection_uuid = "bfa41c64-3720-4cef-88dd-ec795ba36800"
-rp_type = "BGP"
-rp_name = "FCR-Con-BGP"
+connection_uuid    = "bfa41c64-3720-4cef-88dd-ec795ba36800"
+rp_type            = "BGP"
+rp_name            = "FCR-Con-BGP"
 customer_peer_ipv4 = "190.1.1.2"
 customer_peer_ipv6 = "190::1:2"
-customer_asn = "100"
+bgp_enabled_ipv4   = true #Can either be true or false
```

Comment:
Booleans are well understood in the community. Only Enums need to have a comment describing acceptable values. Comment can be removed.